src/yunohost/data_migrations/0011_setup_group_permission.py

```diff
@@ -28,22 +28,42 @@ class MyMigration(Migration):
 
     required = True
 
+    def remove_if_exists(self, target):
+
+        from yunohost.utils.ldap import _get_ldap_interface
+        ldap = _get_ldap_interface()
+
+        try:
+            objects = ldap.search(target + ",dc=yunohost,dc=org")
+        # ldap search will raise an exception if no corresponding object is found >.> ...
+        except Exception as e:
+            logger.debug("%s does not exist, no need to delete it" % target)
+            return
+
+        objects.reverse()
+        for o in objects:
+            for dn in o["dn"]:
+                dn = dn.replace(",dc=yunohost,dc=org", "")
+                logger.debug("Deleting old object %s ..." % dn)
```

Comment:
```suggestion
                logger.debug("Deleting old object '%s' …" % dn)
```

Reply:
I appreciate the energy you put into this but please, we don't need such level of pedanticness for internal comments and debug messages which are not displayed to the regular user...

Reply:
The extraneous effort was writing your comment instead of clicking yes ;)

Non-regular users can also be blind, and then you have to listen to "dot-dot-dot", which gets old. Having non-ellipses in the stringbase is a pain when trying to correct things. If errors are left in comments, they make their way into the other strings sooner or later.